### DISCUSSION TITLE
How to show the GPU memory?

### DISCUSSION THREAD

**Discussion by llagerlof** [fastfetch-cli/fastfetch#1042] [Q&A]
How to show GPU memory and GPU used memory ? I have a NVIDIA card.

**Comment by CarterLi** [comment]
Try `fastfetch -l none --gpu-temp --gpu-driver-specific -s gpu`

**Reply by llagerlof** [reply]
Thank you, but that not worked for me. This is the output of command `fastfetch -l none --gpu-temp --gpu-driver-specific -s gpu`:

```
GPU: NVIDIA GeForce RTX 4070 (5888) @ 3,15 GHz - 33,0°C [Discrete]
```

That GPU has 12GB of memory.

I understand if fastfetch can't do that by default, so, is it possible to inject information into the fastfetch output? I have other means to extract the data I want (gpu memory and used memory), but how to add it to fastfetch output?

**Reply by CarterLi** [reply]
There is a bug that preventing VMEM being shown. Please try the latest dev build. You can download it in the Github Action
